build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -29,6 +29,6 @@
     <PackageReference Include="Microsoft.Build.Framework" Version="$(CLI_MSBuild_Version)" />
     <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="$(PlatformAbstractionsVersion)" />
     <PackageReference Include="Microsoft.DotNet.VersionTools" Version="$(VersionToolsVersion)" />
-    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Feed" Version="1.0.0-prerelease-01929-02" />
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Feed" Version="$(BuildTasksFeedToolVersion)" />
```

Comment:
Don't you still need to do the restore in another context?

Reply:
Line 109 runs some prep work with including a restore: https://github.com/dotnet/cli/blob/08e594b8605f1e60ed17735ec96dba3ea6936ad3/run-build.ps1#L109

We just have to exclude the importation of 'build/Publish.targets' during this call.